test/packages/with-kind/kubernetes/data_stream/state_cronjob/_dev/deploy/k8s/deployment.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       automountServiceAccountToken: true
       containers:
-        - image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.7.0
+        - image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.15.0
```

Comment:
I don't find this definition in the integrations repo for the `kubernetes` package.

Reply:
I think it is applied with Kustomize here https://github.com/elastic/integrations/blob/619d4001668a27da69dd661e2f1300faef3bcccf/packages/kubernetes/_dev/deploy/k8s/kustomization.yaml#L10